README.md

```diff
@@ -58,11 +58,11 @@ You might also prefer the async [higher-level wrapper](https://crates.io/crates/
 nrf-modem = "*"
 ```
 
-## Licence
+## License
 
 Any of the code outside the `./third_party` folder is under the [Blue Oak
-Licence](./LICENCE.md). Any code inside the `./third_party` folder (include
-the Nordic nrfxlib) has its own LICENCE file.
+License](./LICENSE.md). Any code inside the `./third_party` folder (include
+the Nordic nrfxlib) has its own LICENSE file.
```

Comment:
This wasn't a spelling mistake. This repo was started by a Brit and 'licence' is the way you write it in british. Please revert

Reply:
Ok